web/src/app/admin/skills/skills.component.ts

```diff
@@ -50,9 +52,22 @@ export class SkillsComponent implements OnInit {
       .subscribe((data) => {
         if (data) {
           this.skills = data.body;
+          
         }
       });
   }
+  checkDuplicate(enteredSkill){
+    for(let skill of this.skills){
+      if(skill == enteredSkill){
+        this.duplicate = true
+      }
+    }
+    this.duplicate = false
+  }
+  onBlurMethod(){
+    this.skillsModel;
+    this.displayMessage.showWarning("skills name already exist ");
```

Comment:
here use label for displaying message, remove toaster from here

Reply:
done